### DISCUSSION TITLE
How to change order of songs in a playlist?

### DISCUSSION THREAD

**Discussion by HiroDane** [fast4x/RiMusic#2187] [Q&A] [answered]
I am pretty sure I did this before, but I think with new UI changes I cannot figure out how to change the order of songs in an existing playlist. Not as sorting by title or date etc. I need to create a custom order in which they are played. Only option I see is shuffle.

**Comment by Abilomp** [comment]
Just long press the arrow button

**Reply by HiroDane** [reply, marked as answer]
Okay so long pressing the arrow which does sorting shows a new button with a lock icon, After clicking on the lock icon it becomes unlocked and then it allows to rearrange the songs. 
I am bit confused as why is the lock button hidden like that behind a very un-intuitive long hold of the sort arrow. Seems like an unnecessary step. And I would never have guessed that it worked that way. Specially when there is a perfectly functional menu for extra options where this clearly belongs if not available by default.
